Document-Processing/PowerPoint/PowerPoint-Library/NET/Feature-Matrix.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Feature Matrix in .NET PowerPoint Library | Syncfusion
+description: Explore the supported features of Syncfusion® Essential PowerPoint for working with PowerPoint 2007–2016 presentations programmatically.
+platform: document-processing
+control: Presentation
+documentation: UG
+---
+
+# Feature Matrix in .NET PowerPoint Library
+
+Syncfusion® Essential PowerPoint is a [.NET Core PowerPoint library](syncfusion.com/document-processing/powerpoint-framework/net-core) that provides comprehensive support for creating, editing, and managing PowerPoint presentations programmatically. It allows seamless handling of slides, shapes, charts, SmartArt, tables, and more.
```

Comment:
Please modify the intro as,

Syncfusion® Essential PowerPoint is a [.NET PowerPoint library](https://www.syncfusion.com/document-sdk/net-powerpoint-library) that provides comprehensive support for creating, editing, and managing PowerPoint presentations programmatically. It allows seamless handling of slides, shapes, charts, SmartArt, tables, and more.